routes/trading.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const { trade } = require('../controllers/tradingController')
+const { newTrade } = require('../middlewares/validators/trading')
+
+/**
+ * @swagger
+ * /trade:
+ *  patch:
+ *   summary: Used for new trading request
+ *   tags:
+ *     - Trading
+ *   requestBody:
+ *     description: Trading details
+ *     content:
+ *       application/json:
+ *         schema:
+ *           $ref: '#/components/schemas/trading'
+ *   responses:
+ *     200:
+ *       description: Successful trading details
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/tradingSuccess'
+ *     403:
```

Comment:
We are returning 401 from the authenticated middleware.

Reply:
Added this - https://github.com/Real-Dev-Squad/website-backend/pull/222/commits/9ae954a9bd04ab81abdfab5fb8a8b5b5e902aa5f
we will also be returning `403` in one case